### PR TITLE
implement Gaussian

### LINKED ISSUE #99: Implement Gaussian
This issue asks for the support of first-principles (FP) software Gaussian. 
The implementation is supposed to take the advantage of the new interface for adding new FP methods by PR #98

### PR DESCRIPTION
**Pull request by njzjz**
Fix #99.

### CODE REVIEW

File: dpgen2/fp/gaussian.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        command = ' '.join([command, gaussian_input_name])
+        ret, out, err = run_command(command, shell=True)
+        if ret != 0:
+            raise TransientError(
+                'gaussian failed\n',
+                'out msg', out, '\n',
+                'err msg', err, '\n'
+            )                    
+        # convert the output to deepmd/npy format
+        sys = dpdata.LabeledSystem(gaussian_output_name, fmt='gaussian/log')
```
Comment (wanghan-iapcm) [inline dpgen2/fp/gaussian.py:138]: Does `dpdata` assume `nopbc` for gaussian/log?

Comment (njzjz) [reply dpgen2/fp/gaussian.py:138]: Yes, and it also assumes `nopbc` for the input file. However, Gaussian does support the [PBC](https://gaussian.com/pbc/) condition. PBC should be implemented in dpdata.

Comment (njzjz) [reply dpgen2/fp/gaussian.py:138]: https://github.com/deepmodeling/dpdata/pull/397

Comment (wanghan-iapcm) [reply dpgen2/fp/gaussian.py:138]: Great! merged.

Comment (njzjz) [reply dpgen2/fp/gaussian.py:138]: Fixed a bug in https://github.com/deepmodeling/dpdata/pull/401

File: tests/test_prep_run_gaussian.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            'forces': np.zeros((1, 1, 3)),
+            'orig': np.zeros(3),
+            'nopbc': True,
+        })
+        prep_gaussian = PrepGaussian()
+        prep_gaussian.prep_task(
+            conf_frame=system,
+            inputs=inputs,
+        )
+
```
Comment (wanghan-iapcm) [inline tests/test_prep_run_gaussian.py:40]: assert `gaussian_input_name` is generated and in the right place?

**Review by wanghan-iapcm — APPROVED** [review]

**Pull request merged**